docs/datamodel/terminology.rst

```diff
@@ -0,0 +1,72 @@
+.. _ref_datamodel_terminology:
+
+===========
+Terminology
+===========
+
+.. important::
+
+  Below is an overview of EdgeDB's terminology. Use it as a roadmap, but don't
+  worry if it doesn't all make sense immediately. The following pages go into
+  detail on each concept below.
+
+A running EdgeDB process is known as an **instance**. You can create, start,
+stop, and destroy instances on your local computer with the :ref:`EdgeDB CLI
+<ref_cli_overview>`.
+
+.. Instances listen for incoming queries on a connection port.
+
+.. _ref_datamodel_databases:
+
+Each instance can contain several **databases**, each with a unique name. At
+the time of creation, all instances contain a single database called
+``edgedb``.
+
+.. This is the default database; all incoming queries are executed against it
+.. unless otherwise specified.
+
```

Comment:
Do we need to mention that you connect to a specific database of an instance? I think with other databases like MySQL you can connect to the instance and can query any database?

Reply:
Huh this was a hole in my understanding. I don't think this should be mentioned in the Terminology section but I'll add something to this effect on the Connection Parameters page